apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/http/post/PopOff.java

```diff
@@ -54,7 +54,9 @@ public JSONStreamAware processRequest(HttpServletRequest req) {
         boolean keepTransactions = "true".equalsIgnoreCase(req.getParameter("keepTransactions"));
         List<? extends Block> blocks;
         lookupBlockchainProcessor();
+        lookupDexOrderProcessor();
```

Comment:
Better to suspend it by event emitted from blockchainProcessor

Reply:
done